THIRD-PARTY-NOTICE.txt

```diff
@@ -0,0 +1,7765 @@
+Azure SDK for Java uses third-party libraries or other resources that may be
+distributed under licenses different than the Azure SDK for Java software.
+
+In the event that we accidentally failed to list a required notice, please
+bring it to our attention. Post an issue or email us:
+
+           azjavasdkhelp@microsoft.com
+
+The attached notices are provided for information only.
+
+License notice for Azure SDK for Java
+-------------------------------
+
+Copyright (c) Microsoft. All rights reserved.
+Licensed under The MIT License (MIT)
+
+Available at
+https://github.com/Azure/azure-sdk-for-java/blob/master/LICENSE.txt
+
+License notice for org.eclipse.jetty:jetty-http:9.4.11.v20180605
+------------------------------------------------------------------------------
+
+                                 Apache License
+                           Version 2.0, January 2004
+                        http://www.apache.org/licenses/
```

Comment:
We discussed how to do apache after you asked about it, and I pointed you the correct way to do it. This wasn't it - we should just link out to the text rather than reproduce it.

Reply:
You asked me to check the example you gave to me. They put the text over instead of link. Some of the dependency put the text like this exactly in their license. Do we want to paste the link directly?